app/static/js/eventKiosk.js

```diff
@@ -61,8 +61,20 @@ function submitData(hitEnter = false){
         "eventid": $("#eventid").val(),
         "bNumber": $("#submitScannerData").val()
       },
-      success: function(flasherMessage) {
-        eventFlasher(flasherMessage, "success");
+      success: function(result) {
+        flasherStatus = "success"
+        if (status === "already in") {
+          message = result.user + " Already Signed In!"
+        } else if (status === "banned") {
+          message = result.user + " is Banned."
+          flasherStatus = "danger"
+        } else if (status === "does not exist") {
+          message = "User does not exist"
+          flasherStatus = "danger"
+        } else {
+          message = result.user + " Successfully Signed In!"
+        }
+        eventFlasher(message, flasherStatus);
```

Comment:
This code does not work, no matter what the input is, whether the user does not exist or the user has already signed in, it still says "Successfully Signed In!"

Reply:
Fixed. idk how that happened